pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -271,9 +270,7 @@ stdenv.mkDerivation rec {
     gperf
     ninja
     meson
-    coreutils # meson calls date, stat etc.
```

Comment:
Was initially introduced in https://github.com/NixOS/nixpkgs/commit/3240b48f7468990457b6975c79f0e3e26316bbae#diff-a284772b505bd573f834b58591d380128545257d9e02601dbe112548526c8710R79-R82 and is still required in general. The usage here was added by myself in https://github.com/NixOS/nixpkgs/commit/eb862c48ddc288243c5447e8decc623b10ef165c#diff-a284772b505bd573f834b58591d380128545257d9e02601dbe112548526c8710R34 as apparently it wasn't part of stdenv back then? Can you please provide details why we don't need it anymore? IMHO this is part of the work of proposing such a change. Give me a explanation not just a description of the action taken. If it still compiles it is even more curious to figure out why it was needed initially.

Reply:
It was even back then part of the stdenv https://github.com/NixOS/nixpkgs/blob/eb862c48ddc288243c5447e8decc623b10ef165c/pkgs/stdenv/common-path.nix#L2